pkg/conf/conf.go

```diff
@@ -448,6 +453,24 @@ type Pollers struct {
 	namesInOrder []string
 }
 
+var defaultTemplate = &[]string{"default.yaml"}
```

Comment:
let's make this line into:
`var defaultTemplate &[]string{"default.yaml", "custom.yaml"}`
then delete lines  495-497 from poller.go